### ISSUE TITLE
find symbol gets confused after rename file or dir

### ISSUE THREAD

**Issue by benedekfazekas** [clojure-emacs/refactor-nrepl#91]
if in namespace A i define a symbol and refer and use it in namespace B find usages returns 2 occurrences one for the definition of the symbol and one for the usage. If i rename namespace A (or the directory it lives in) find usages only returns one of the occurrences depending on where you call it from: only returns the definition of the symbol if called from renamed namespace A and only returns the usage if you call it from namespace B

**Comment by expez** [comment]
Didn't you fix this the other day by rebuilding the ast in more places?

**Comment by benedekfazekas** [comment]
good question. will retest

**Comment by benedekfazekas** [comment]
still can reproduce

**Comment by benedekfazekas** [comment]
this seems to be the core of the problem:

after renaming `energy-pricing.greeter` => `energy-pricing.newgreeter`
```clojure
(ns-resolve (find-ns 'energy-pricing.dependent) 'greet)
;;=> #'energy-pricing.greeter/greet
```
it should return:
```clojure
;;=> #'energy-pricing.newgreeter/greet
```

**Comment by expez** [comment]
It's temping to call `cider-refresh` after this op, then.  The only caveat is that I've experienced `tools.namespace` getting confused when reloading files that have been moved.  I'm not sure this is still an issue, but perhaps it would be better, although slower, to also a load the file when we're visiting to fix the indentation of the ns-form.

**Comment by benedekfazekas** [comment]
fixed via 9bc39148e82e0ee489790ee84ae34abcfb7e473d
